### PR TITLE
Add basic stats collection using probe.gl's Stats class

### PR DESCRIPTION
**Pull request by ibgreen**
#### Background
- For #1530
- Generic stats collection system. Collected stats can be presented using various mechanisms:
- currently animation loop logs stats using probe.gl's `Log.table()`
- In the future, Seer integration and Stats Widgets will be added as alternative presentation mechanisms.
#### Change List
- `Deck` class creates a `Stats` instance and passes it around
- other classes add timings

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -20,6 +20,7 @@ addons:
     - libxi-dev
 node_js:
   - 6.9.1
+  - 8
```
Comment (jianhuang01) [inline .travis.yml]: what's this?

Comment (balthazar) [reply .travis.yml]: Adding another set of tests on Travis for node v8

Comment (ibgreen) [reply .travis.yml]: Which node versions Travis CI should test on. 6 and 8 are the latest LTS (long-term support) versions, so good to test on those.

Comment (ibgreen) [reply .travis.yml]: If you click on the travis CI link below you can see there are multiple test runs.

**Pull request merged**